getSelects.php

```diff
@@ -0,0 +1,54 @@
+<?php
+    //This file is for getting user information for switching session event data such as location, time and day. 
+    //pdo.php contains the pdo variable.
+    include './pdo.php';
+    //Gets possible day selections
+    echo "<h2>Change Session:</h2><br>";
+    echo "<div id='dayselectdiv'>";
+        echo "<h3>Select Day:</h3>";
+        //Gets the days to select from. The form attribute allows for drop down menu information to be passed when user submits data.
+        echo "<select id='sessionselect' name='sessionday' form='setEventForm'>";
+            echo "<option value='saturday'>Saturday</option>";
+            echo "<option value='sunday'>Sunday</option>";
+        echo "</select><br>";
+    echo "</div>";
+
+    //Gets the possible rooms to select from";
+    echo "<div id='roomselectdiv'>";
+        echo "<h3>Rooms:</h3>";
+        echo "<select id='roomselect' name='sessionroom' form='setEventForm'>";
+            for($i = 0; $i <= 4; $i++){ //assuming 4 floors
+                for($j = 0; $j < 10; $j++){ //assuming 10 rooms on each floor
+                    $rm = "$i" ."0" . "$j";
+                    echo "<option value=" . $rm. ">" . $rm . "</option>";
+                }
+            }
+        echo "</select><br>";
+    echo "</div>";
+    //get the possible times to select from
+    echo "<div id='divtimeselect'>";
+        echo "<h3>Time:</h3>";
```

Comment:
Maybe we can clarify here that we're specifying the start time and that it's meant to last an hour long?